MatrixKit/Views/ReadReceipts/MXKReadReceiptTableViewCell.m

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2017 Aram Sargsyan
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MXKReadReceiptTableViewCell.h"
+
+
+@implementation MXKReadReceiptTableViewCell
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+    self.avatarImageView.enableInMemoryCache = YES;
+    NSMutableAttributedString *readString = [[NSMutableAttributedString alloc] initWithAttributedString:self.readLabel.attributedText];
+    [readString setAttributes:@{NSFontAttributeName:[UIFont boldSystemFontOfSize:15]} range:NSMakeRange(0, readString.length)];
+    self.readLabel.attributedText = readString;
```

Comment:
I advised you to use an attributed string to display the receipt details in only one UILabel instance (instead of using 2 labels:`readLabel` and `receiptDescriptionLabel`).

If you decide to keep this label named `readLabel` for the constant prefix "Read:", you don't have to use attributed string. And you may define the text font in the xib file.
About the label text value, you have to add a new key-value in the MatrixKit.strings file to handle correctly the translation (for example "receipt_status_read" = "Read:";). Then you use the key to define the label text:
`self.readLabel.text = [NSBundle mxk_localizedStringForKey:@"receipt_status_read"]`.

Personally I would prefer to remove the `readLabel` object. We should let the owner of the MXKReadReceiptTableViewCell instance set an attributed string in the `receiptDescriptionLabel` label to include the receipt status ("Read:" or something else). In that case the attributed string will be handled at the application level (in Riot-ios source code for example). CAUTION: the string translation will be handled at application level (see Vector.strings in Riot-ios ressources).